js/private/js_expand_template.js

```diff
@@ -0,0 +1,87 @@
+const fs = require('fs')
+
+/**
+ * The status files are expected to look like
+ * BUILD_SCM_HASH 83c699db39cfd74526cdf9bebb75aa6f122908bb
+ * BUILD_SCM_LOCAL_CHANGES true
+ * STABLE_BUILD_SCM_VERSION 6.0.0-beta.6+12.sha-83c699d.with-local-changes
+ * BUILD_TIMESTAMP 1520021990506
+ *
+ * Parsing regex is created based on Bazel documentation describing the status file schema:
+ *   The key names can be anything but they may only use upper case letters and underscores. The
+ *   first space after the key name separates it from the value. The value is the rest of the line
+ *   (including additional whitespace).
+ */
+function _parseStatusFile(p) {
+    if (!p) return []
```

Comment:
Should this not be `{}`?

I assume that means this `if` is never hit and we can just delete the line?

Reply:
Yea; it can be removed; guess that code is wrong in rules_nodejs where I borrowed it from

Reply:
✅